pull.go

```diff
@@ -141,9 +146,53 @@ func localImageNameForReference(ctx context.Context, store storage.Store, srcRef
 }
 
 // Pull copies the contents of the image from somewhere else to local storage.
-func Pull(ctx context.Context, imageName string, options PullOptions) (types.ImageReference, error) {
+func Pull(ctx context.Context, imageName string, options PullOptions) error {
```

Comment:
I haven't gone through this throughly, but we need to be careful altering this function as the FROM command also uses it.  Please verify that FROM is behaving as expected with this changes, especially so in the tests.

Reply:
I noticed that the other commands eventually call the [`pullImage`](https://github.com/containers/buildah/pull/1263/files#diff-27c9dec751c416fdf10e5e5e780b008cR205) function, so modifying this function will have no effect on other commands.

Reply:
Oh you're right @q384566678 , I had it my head that From used this too, but this is used only by Pull, good call.